lib/main_production.dart

```diff
@@ -1,9 +1,7 @@
-import 'package:flutter/material.dart';
-
 import 'flavors.dart';
-import 'main.dart';
+import 'main.dart' as app;
 
 void main() {
   F.appFlavor = Flavor.production;
-  runApp(const App());
+  app.main();
 }
```

Comment:
Using flutter_config, we can get rid of duplicating these app launches. We can provide the env and run by command like this:

```
$ fvm flutter run --flavor production
```